### PR TITLE
feat(install_panels): enable ntp server after settings

### PR DESCRIPTION
**Pull request by FrankYang0529**
issue: https://github.com/harvester/harvester/issues/935

Enable NTP time sync if NTP servers can be connected.

### CODE REVIEW

File: pkg/console/util.go [hunk 1]
```diff
@@ -130,6 +130,31 @@ func validateNTPServers(ntpServerList []string) error {
...
+
+	ntpServersString := fmt.Sprintf("NTP=%s\n", strings.Join(ntpServerList, " "))
+	_, err = file.WriteString(ntpServersString)
+	if err != nil {
+		return err
+	}
+
+	_, err = exec.Command("timedatectl", "set-ntp", "true").Output()
+	if err != nil {
+		return err
```
Comment (gitlawr) [inline pkg/console/util.go:152]: Use CombinedOutput() and log the output when we hit an error. The returned error is usually an ExitError that may not have much info for debugging

Comment (FrankYang0529) [reply pkg/console/util.go]: Updated. Thank you.

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**